### ISSUE TITLE
nvda does not announce the name of the object in focus when opening a dialog window in libre office 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13068]
### Steps to reproduce:
open the writer as an example.
type a text
select this text
in format menu, press enter to open character
the character window will open 
### Actual behavior:
nvda informs dialog character
but it does not inform the name and type of the focused control, in this case an edit field.
the field is in focus because even nvda does not announce changes in this edit field, such as deleting or typing characters changes are made in the edit field.
when taking focus out of the edit field by pressing tabe as an example and then shift + tabe nvda correctly announces the control
This is possibly present in other programs and dialogs in libre office. 
### Expected behavior:
nvda should announce the name of the focus control, as well as the control type, and changes to the control, without requiring the user to remove and return focus to the control. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-24222,6413fc4a

#### Windows version:
11 22000.346
#### Name and version of other software in use when reproducing the issue:
writer soffice.bin, LibreOffice 7.3.0.0

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log
[libreofficedialogo.txt](https://github.com/nvaccess/nvda/files/7564774/libreofficedialogo.txt)

**Comment by Adriani90** [comment]
cc: @michaelweghorn

**Comment by michaelweghorn** [comment]
I can reproduce with current development versions of NVDA (d2d4bfce2d31de8fca3c6df16caf93350b26374d) and LibreOffice (c379f1b978c3ad4578c130675c1e0a82c70823c0). This probably needs further analysis on LibreOffice side.

**Comment by michaelweghorn** [comment]
Corresponding LibreOffice bug report: https://bugs.documentfoundation.org/show_bug.cgi?id=161962

**Comment by Adriani90** [comment]
@michaelweghorn I have commented on the LO issue tracker as well, asking whether there is any progress in newer LO versions?

**Comment by michaelweghorn** [comment]
> [@michaelweghorn](https://github.com/michaelweghorn) I have commented on the LO issue tracker as well, asking whether there is any progress in newer LO versions?

This is still unchanged in the current development version from what I can see.

**Comment by Adriani90** [comment]
This is still reproducible in Libreoffice 26.2 last daily build with NVDA 2025.3.
@michaelweghorn is the keyboard focus set on a control at all automatically when opening a dialog in LO? Or is it in general also for sighted people that keyboard focus is not set and the user has to press tab to trigger it for the first time?

**Comment by michaelweghorn** [comment]
> This is still reproducible in Libreoffice 26.2 last daily build with NVDA 2025.3. [@michaelweghorn](https://github.com/michaelweghorn) is the keyboard focus set on a control at all automatically when opening a dialog in LO? Or is it in general also for sighted people that keyboard focus is not set and the user has to press tab to trigger it for the first time?

@Adriani90 Keyboard focus is on a control when a dialog opens. The problem is that it's not announced by NVDA/screen readers, possibly due to a missing a11y focus event.
